bool calibrate
int32 chan
---
bool moveSuccess
